docs/reference/rest-api/usage.asciidoc

```diff
@@ -254,7 +254,18 @@ GET /_xpack/usage
         "total" : 0,
         "failed" : 0
      }
-   }
+   },
+    "functions": {
+      "ADD" : 2,
+      "EQUALS" : 2,
+      "MAX" : 1,
```

Comment:
This it problematic: this list is dynamic and potentially very long. It depends on the functions that are used in queries.